go.mod

```diff
@@ -75,10 +74,10 @@ require (
 	github.com/prometheus/common v0.6.0
 	github.com/prometheus/procfs v0.0.3 // indirect
 	github.com/satori/go.uuid v1.2.0
-	github.com/spf13/cast v1.2.0
-	github.com/spf13/cobra v0.0.3
+	github.com/spf13/cast v1.3.0
+	github.com/spf13/cobra v0.0.5
```

Comment:
hmm this roles cobra and vipers versions, what's the driver behind that?

Reply:
i only updated the cobra version, viper is picked up by go mod vendor and go mod tidy